app/renderer/actions/Inspector.js

```diff
@@ -152,30 +175,53 @@ export function unselectHoveredElement (path) {
  */
 export function applyClientMethod (params) {
   return async (dispatch, getState) => {
-    const isRecording = params.methodName !== 'quit' &&
-                      params.methodName !== 'getPageSource' &&
-                      getState().inspector.isRecording;
+    const isRecording =
+      params.methodName !== 'quit' &&
+      params.methodName !== 'getPageSource' &&
+      getState().inspector.isRecording;
     try {
-      dispatch({type: METHOD_CALL_REQUESTED});
+      dispatch({ type: METHOD_CALL_REQUESTED });
       const callAction = callClientMethod(params);
-      const {contexts, contextsError, currentContext, currentContextError,
-             source, screenshot, windowSize, result, sourceError,
-             screenshotError, windowSizeError, variableName,
-             variableIndex, strategy, selector} = await callAction(dispatch, getState);
+      const {
```

Comment:
I understand this is the work of an autoformatter, but it really makes it super hard to review this PR, because it is not clear what changes are new and what was just formatting stuff. Would it be possible to revert autoformatting changes to simplify the review process and to reduce the cognitive load?

Reply:
I also understand that, it's pretty hard to revert it and the problem/challenge is that we don't have real editor format rules.
I can check what is possible, but it might be reverted all the time I store the file :(